core/sql/parser/ElemDDLColDef.h

```diff
@@ -309,6 +312,8 @@ class ElemDDLColDef : public ElemDDLNode
   NABoolean seabaseSerialized_;
 
   NABoolean isColDefaultSpec_;
+
+  Int32 errCode_;
```

Comment:
This is leftover code, I will remove it. When there are more comments, and change together